.github/workflows/python-service.yml

```diff
@@ -184,6 +184,7 @@ jobs:
           check_artifacts: true
           search_artifacts: true
           if_no_artifact_found: warn
+          workflow_conclusion: ""
```

Comment:
The default is success, but in case we want to force the merge, the build will fail due to the diff, but still, we want that the next coverage percentage points to the forced one.